devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -64,25 +76,17 @@ metadata {
 		details(["motion", "battery", "tamper"])
 	}
 
-	// Preferences for Everspring SP815 and Everspring SP817
-	// preferences {
+	// Preferences for  Everspring SP817
+	preferences {
 		section {
 			input(
-					title: "Settings Available For Everspring SP815 and SP817 only",
-					description: "",
+					title: "Settings Available For Everspring SP817 only",
+					description: "To activate device's settings changes press the tamper switch on the device three times or check the device manual.",
```

Comment:
Let's fix the first few words to:

To apply updated device settings to the device press the tamper switch on the device three times or check the device manual.

Reply:
sure, done